website/data/plugin-framework-nav-data.json

```
@@ -7,8 +7,21 @@
     "path": ""
   },
   {
-    "title": "Learn Collection",
-    "href": "https://learn.hashicorp.com/collections/terraform/providers-plugin-framework?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS"
+    "title": "Getting Started",
+    "routes": [
+      {
+        "title": "Code Walkthrough",
+        "path": "getting-started/code-walkthrough"
+      },
+      {
+        "title": "Tutorials",
+        "href": "https://learn.hashicorp.com/collections/terraform/providers-plugin-framework?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS"
+      },
+      {
+        "title": "Terraform Provider Scaffolding",
```

Not so sure about whether this might be better

```suggestion
        "title": "Clone Template Repository",
```